orchestration/dagster_orchestration/hca_manage/tests/test_dataset_manager.py

```diff
@@ -52,3 +52,13 @@ def test_create_dataset_with_policy(self):
 
         self.manager.data_repo_client.create_dataset.assert_called_once()
         self.assertEqual(self.manager.data_repo_client.add_dataset_policy_member.call_count, 2)
+
+    def test_validate_dataset_name(self):
+        with self.assertRaises(ValueError):
+            validate_dataset_name(
+                "fake_dataset_name"
+            )
+
+        validate_dataset_name(
```

Comment:
⛏️ These should be separate tests, one for validation for the success path, one for the failing path (i.e, a unit test should focus on testing one thing only)

Reply:
made two separate tests